java/core/src/test/java/com/microsoft/bond/testmodel/MyEnum.java

```diff
@@ -78,6 +78,30 @@ public static MyEnum get(int value) {
         }
     }
 
+    public static MyEnum parse(String str) {
```

Comment:
I prefer we keep it close to true Java enums until we get some constructive feedback from customers on this.

---
In reply to: [132561015](https://github.com/Microsoft/bond/pull/552#discussion_r132561015) [](ancestors = 132561015)